src/parser/jobs/brd/modules/Util.tsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * @author Yumiya
+ * @author Ririan
+ */
+
+import {DamageEvent, Event} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Combatants from 'parser/core/modules/Combatants'
+import Downtime from 'parser/core/modules/Downtime'
+import Enemies from 'parser/core/modules/Enemies'
+import Invuln from 'parser/core/modules/Invulnerability'
+import Timeline from 'parser/core/modules/Timeline'
+import React from 'react'
+import {Button} from 'semantic-ui-react'
+import AdditionalStats from './AdditionalStats'
+
+const CONVERSION_FACTOR = 0.1
+
+const DHIT_MOD = 1.25
+
+const TRAIT_STRENGTH = 0.20
+
+interface StatusEffect {
+	id: number
+	name: string
+	icon: string
+	duration?: number
+	amount?: number
```

Comment:
I can't see `amount` being used anywhere from StatusEffect. Core has a status type (from `src/data/STATUSES/STATUSES.d.ts`) which can be imported with `import {Status} from 'src/data/STATUSES'`. It's otherwise an identical interface.